lms/djangoapps/bulk_email/tests/test_models.py

```diff
@@ -60,6 +68,20 @@ def test_bad_to_option(self):
         with self.assertRaises(ValueError):
             CourseEmail.create(course_id, sender, to_option, subject, html_message)
 
+    def test_cohort_target(self):
+        course_id = SlashSeparatedCourseKey('abc', '123', 'doremi')
```

Comment:
Huh, I had no idea about that. Is that a safe drop-in replacement everywhere? I'll just go ahead and update all the references in this file.